docker/test-tls/nomad-server-config/server.hcl

```diff
@@ -0,0 +1,12 @@
+tls {
```

Comment:
Could you clarify why we need this file?

Reply:
https://www.nomadproject.io/docs/configuration/tls.html

Reply:
COPY ./nomad-server-config/ /etc/nomad.d/